src/components/Slogan/Slogan.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Slogan.css';
+
+
+export default function Slogan({
+  
+  slogan,
+  setSlogan,
+  sloganArray,
+  setSloganArray,
+
+}) {
+  const logSlogan = () => {
+    setSlogan('');
+    setSloganArray((prevState) => [...prevState, slogan]);
+
+  };
```

Comment:
Great job utilizing prevState in your setState function. I would although recommend switching lines 14 and 15. I know we talked about it in the metal rooms, I have run into the issue of setting my variable as an empty string before the variable was placed into a setter function, it also helps with readability if someone is to look at your code. 👍